.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,8 +12,8 @@ Make sure these boxes are checked before your pull request (PR) is ready to be r
 * [ ] Insert-step functionality is working correct as expected.
 > We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!
```

Comment:
I suggest this should be commented too. Would you like to do it?

Reply:
Done! @harshkhandeparkar